docs/guides/src/main/server/importExport.adoc

```diff
@@ -0,0 +1,101 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+
+<@tmpl.guide
+    title="Importing and Exporting Realms"
+    summary="An overview about how to import and export realms">
+
+In this guide, you are going to understand the different approaches for importing and exporting realms using JSON files.
+
+== Exporting a Realm to a Directory
+
+To export a realm, you can use the `export` command.
```

Comment:
```suggestion
To export a realm, you can use the `export` command. Your Keycloak server instance must not be started when invoking this command.
```

Reply:
Just to make sure people don't try to run the export command in e.g. another shell instance while running keycloak. Just tried it myself, then the export command fails (was expected from my side).